test/spec/ProjectModel-test.js

```diff
@@ -732,13 +732,41 @@ define(function (require, exports, module) {
                 });
                 
                 it("adjusts the selection if the renamed file was selected", function () {
+                    spyOn(model, "_renameItem").andReturn(new $.Deferred().resolve().promise());
```

Comment:
This wasn't necessary before because the selection update would happen even if the rename failed (since the files don't actually exist). In general, it would probably be a good idea to either add a mock filesystem to these tests, or always stub out `_renameItem` and any other functions that access the real filesystem.